backend/src/packages/auth/auth.controller.ts

```diff
@@ -238,6 +249,15 @@ class AuthController extends BaseController {
       payload: await this.authService.getAuthenticatedUser(options.user.id),
     };
   }
+
+  private async deleteAuthenticatedUser(
```

Comment:
we need to add this route to swagger

Reply:
I implemented the other comments, still working on this one.

Reply:
Done✅